Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+\section{Processi di supporto}
+\subsection{Processo di documentazione}
+\subsubsection{Scopo}
+L'obiettivo proposto dalla sezione è quello di prendere nota delle norme che andranno a regolare la stesura della documentazione durante il ciclo di vita del software, così da ottenere documenti formalmente concordi.
+\subsubsection{Descrizione}
+La sezione contiene le norme tramite le quali avviene una corretta stesura dei documenti, comprendente verifica e approvazione. I membri del gruppo sono tenuti ad attenersi alla suddetta sezione.
+\subsubsection{Aspettative}
+Le aspettative riguardanti il processo in questione sono:
+\begin{itemize}
+	\item l'identificazione di una struttura ben organizzata comune ai prodotti del processo in questione;
+	\item la raccolta di norme da rispettare durante la stesura dei documenti.
+\end{itemize}
+\subsubsection{Ciclo di vita di un documento}
+Ogni documento redatto passa per le seguenti fasi:
+\begin{itemize}
+	\item \textbf{Creazione} del documento tramite template;
+	\item \textbf{Strutturazione} del documento tramite registro delle modifiche,  indice di contenuti e/o indice di figure e tabelle;
+	\item \textbf{Stesura} del corpo del documento,  scritto da più membri del gruppo;
+	\item \textbf{Revisione} del documento, gestita da un \textit{Verificatore} che,  regolarmente,  riesamina le sezioni del suddetto;
+	\item \textbf{Approvazione} del documento, stabilita dal \textit{Responsabile di Progetto} quando la revisione risulta positiva; consente il rilascio del documento.
```

Comment:
Cambiare
"... \textit{Responsabile di progetto}"
in
"... \RdP{} ..."

Reply:
-Il punto finale va sostituito anche se si tratta dell'ultimo elemento dell'elenco puntato?

-togliere il ; non cambia il senso alla frase?